content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -82,7 +82,7 @@ installation instructions. The list does not try to be exhaustive.
 * [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA
   networking solution for Kubernetes. Spiderpool is supported on bare metal, virtual machines,
   and public cloud environments.
-* [Weave Net](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/)
+* [Weave Net](https://github.com/weaveworks/weave )
```

Comment:
Minor nit: There seems to be an extra space at the end of the link which could be easily removed.
```suggestion
* [Weave Net](https://github.com/weaveworks/weave)
```